### ISSUE TITLE
Missing description and examples when using Pass Through security scheme

### ISSUE THREAD

**Issue by bennidi** [mulesoft/api-console#403]
We are using JWT as authorization mechanism across our APIs. I have set it up following [this example](https://github.com/mulesoft/api-console/blob/master/test/regression/assets/raml/security-schema-pass-though.raml).

```

jwt:
  description: |
    Authorization with JWT Token passed as authorization header. See Headers section.
  type: Pass Through
  describedBy:
    responses:
      401:
        description: |
          Bad or expired token

       
      403:
        description: |
          Access denied due to invalid access rights
        body:
          application/json:
            example:
              value:
                status : denied

```

And I use a trait to mark routes using JWT.

```

headers:
  authorization:
    type: string
    description: |
        The base-64 encoded JWT token containing the uuid (sub) and roles of the requesting user. 
    required : true
    example: dsad.asdsa.dasdasd
securedBy: [jwt]

```

This is very handy because the console allows Pass Through and the example value of the Authorization header is used to prepopulate the header field.

The problem is that in the response section the 401 and 403 are listed but empty. They do not contain the description or examples as given in the security scheme description.

**Comment by bennidi** [comment]
Writing this issue made me play around with it a bit more and I found a solution that I like. However, I still think that the behavior I am seeing with the above configuration is a bug.

My solution for now is to move the responses section from the security scheme to the trait. This even allows to ad more examples within a route that implements this trait. Full example:

```

jwt:
  description: |
    Authorization with JWT Token passed as `authorization` header. See Headers section.
  type: Pass Through
  describedBy:
    responses:
      401:
        description: |
          Bad or expired token
      403:
        description: |
          Access denied due to invalid access rights



headers:
  authorization:
    type: string
    description: |
        The base-64 encoded JWT token containing the uuid (sub) and roles of the requesting user. A token can be obtained from auth-services endpoint using either **api.xxxx.io** or **staging.api.xxxxx.io** as base URL. Example:

            $ curl 'https://staging-api.xxxxxx.io/auth/login' -H \
            'X-Locale: de_DE' -H 'Content-Type: application/json; charset=UTF-8' -H \
            'Connection: gent: okhttp/3.4.2' --data-binary \
             '{"password":"xxxxx","username":"xxxxx@c.vb"}'
    required : true
    example: eyJhbGciOiJTSEEtSGFtZHVsbGFoIn0=.eyJzdWIiOiI0MjUwMzM5Yy01YTZhLTQ5ZT gtOGJhOS1kOTdiNzIzZGRhMmEiLCJsb2NhbGUiOiJkZV9ERSIsIn JvbGVzIjpbIlJPTEVfVVNFUl9CQVNJQyJdfQ==.mumboJumbo
securedBy: [jwt]
responses:
  401:
    description: |
      Bad or expired token

      ```json

      {
        "message": "Bad or expired token"
      }

      ```
  403:
    description: |
      Access denied due to invalid access rights

      ```json

      {
        "message": "Access denied"
      }

      ```
    body:
      application/json:
        examples:
          access-denied:
            reason : 'bad token'



/wallet:
  /me:
    get:
      is: [ user-segmented, jwt-authorized ]

```

**Comment by carowright** [comment]
Fix available in [v3.0.17](https://github.com/mulesoft/api-console/releases/tag/v3.0.17)
